components/NumericInput/NumericInput.vue

```diff
@@ -278,13 +278,22 @@ export default {
       return this.valueAsString == null || this.valueAsString.length == 0;
     },
     disableSmallDec() {
-      return this.numericValue - this.incDecValues[0] < this.minValue;
+      return (
+        this.numericValue - this.incDecValues[0] < this.minValue ||
+        isNaN(this.numericValue)
```

Comment:
Now, assuming that a negative value might be entered. Then, if `Value` is currently blank (Nan) and `minValue` is a negative number then the decrement buttons that don't subceed `minValue` will remain enabled. Otherwise, the decrement buttons remain disabled. This covers cases where the user wants to enter a negative number and the `minValue `allows it and the more traditional case where the user wants to clear the field before entering a value.